src/AutomationTests/SnapshotCommandUnitTests.cs

```diff
@@ -203,12 +145,14 @@ public void Execute_UnableToSetTestModeDataContext_ThrowsAutomationException_Err
                     POIElementContextGet = () => new ElementContext(CreateA11yElement()),
                 };
 
-                InitializeShims(populateLocationHelper: false, enableRetention: true, shimTargetElementLocator: true,
+                InitializeShims(shimTargetElementLocator: true,
                     shimUiFramework: true, setTestModeSucceeds: false);
 
+                var config = Config.Builder.ForProcessId(-1).Build();
+
                 try
                 {
-                    SnapshotCommandResult result = SnapshotCommand.Execute(new Dictionary<string, string>());
+                    SnapshotCommandResult result = SnapshotCommand.Execute(config, OutputFileHelperStub);
                 }
                 catch (System.Exception ex)
```

Comment:
"System." is not needed here

```suggestion
                catch (Exception ex)
```